pygmt/clib/session.py

```diff
@@ -1946,6 +1951,70 @@ def virtualfile_to_dataset(
             return result.to_numpy()
         return result  # pandas.DataFrame output
 
+    def virtualfile_to_raster(
+        self,
+        vfname: str,
+        kind: Literal["grid", "image", "cube", None] = None,
```

Comment:
Perhaps it makes more sense to let `kind` default to `grid` instead, since very few modules can output `image`/`cube` anyway.

Reply:
Oh yes, that's a lot smarter actually.